cognito-user-pools/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,16 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Amazon Cognito Identity Provider"
```

Comment:
Shouldn't the name include "user pools"?

Reply:
Fixed